src/intentHandlers/ProjectStatus.js

```diff
@@ -269,6 +270,116 @@ const newlyCreated = async ((kanbanBoardID, StatusIDs, middleTS, oldestTS, subQu
     return newlyCreated;
 });
 
+// Checks if there are any red columns and returns array of objects consisting of the results
+const redColumnCheck = async((kanbanBoardID, boardConfig) => {
+    const allOk = [{
+        "text": `Kanban board Columns are all healthy! :white_check_mark:`,
+        "color":"good",
+        "mrkdwn_in": ["text"]
+    }];
+    const columns = boardConfig.columnConfig.columns;
+    const getStatusIDs = function (column) {
+        var statusIDs;
+        if (column.name.toUpperCase() !== 'BACKLOG') {
+            statusIDs = fp.map('id')(column.statuses);
+        } else {
+            return null;
+        }
+        return [column, statusIDs];
+    };
+    const checkColumn = function (columnstatusIDs) {
+        if (columnstatusIDs != null){
+            column = columnstatusIDs[0];
+            statusIDs = columnstatusIDs[1];
+            var jiraResponse = await(JiraService.boardInfo(`${kanbanBoardID}/issue?jql=status in (${statusIDs.join(',')}) and issueType!= Epic and resolution is EMPTY ORDER BY created DESC`)
```

Comment:
In the future, consider separating impure, fallible, stafeful calls like this from pure functions.